submissions/GeometricalMLforNeuroscience.md

```diff
@@ -0,0 +1,215 @@
+ 
+Remarkably, the machine learning community is rapidly advancing the understanding of the training of class of artificial neural network. For example, the loss landscape emerging from the optimization of regularized linear autoencoder has been discovered. [28]
+  
+The nonlinear dynamics of deep linear neural networks was explicitly found as soon as 2014. [29]  Since then, this line of reasoning was modified and extended to start capturing the learning trajectory of recurrent neural network. [30] Very interestingly, the modification of the weight matrix was observed to be itself low-dimensional for recurrent neural network trained on simple task. This observation complements what was observed in vivo with BCI interface. It will be interesting in the future to train RNN on more complex tasks while monitoring the dimensionality of changes of synaptic weights.
+  
+The progress made using tools from statistical physics to the understanding of network performances has grown recently grown faster. At its core techniques lie the use of tools from random matrix theory (the study of matrix where each element comes from a certain random distribution) and result from probability theory. A recent paper providing new results as well as a good summary of the literature was recently released and focus on an analytical expression of the loss of teacher-student models. [31]
+ 
+More generally, the use of random matrix theory is instrumental in neuroscience. For example, it allows to have an estimate of the number of principal components that accounts for true feature of the data rather than noise. [32]
+
+The theoretical work on learning is vast, we will therefore restrict our analysis of the literature by considering cases were theoretical tools make use of the geometrical properties of the measured neuronal data. In doing so we would like to highlight how differential geometry may serve as an important toolbox for theorist and for experimentalists.
+
+
+# Differential Geometry: theoretical tools
+
+Differential geometry gathers a set of mathematical tools to study manifolds: how can we define maps from one manifold to another? How can we study the propagation of a wave on a manifold? This toolbox was extensively used to formulate physical theory.  In neuroscience, its importance was stressed as it helped to provide an understanding of the nature of visual hallucinations.  [33]
+ 
+More recently, such tools were used to justify theoretically that computations of deep neural network could transform a complex, ie curved, manifolds (inputs) into a simpler, flat, one [34]
```

Comment:
Interesting, but how exactly are such transformations relevant to understanding brain computation?  Does this have to do with invariance to certain input transformations?

Reply:
Hi Nicolas, thank you for your great feedback, I will update the submission tonight to take it into account! Concerning your question: let me take a simple example, let us say you wish to understand how space coding emerge. From electrophysiology , i.e, brain recording, we know that space is coded over a torus (a donuts) in some part of the mice brain (the medial entorhinal cortex MEC, see grid cells). We also know that the head-direction is coded over a ring. Intuitively, because the HD circuit projects to the MEC, we therefore expect this ring to be combined to another ring so that the donut is effectively generated. In some sense, we need to combine two manifold to produce a new one, the product of the two ring. Differential geometry provides the tool to help understand how this can be achieved by two neural networks converging into a recurrent one in MEC. It also has to do with invariant: symmetries are describe by group. The intuition here is that the group of each ring will be merged to generate a group for the torus, with two actions.